app/src/main/java/com/framgia/fsalon/screen/scheduler/SchedulerPresenter.java

```diff
@@ -25,26 +48,58 @@ public void onStart() {
 
     @Override
     public void onStop() {
+        mCompositeDisposable.clear();
     }
 
     @Override
-    public void getSchedulers() {
-        List<SchedulerSection> sections = new ArrayList<>();
-        List<Scheduler> schedulers = new ArrayList<>();
-        String title = new String("12-07-2017");
-        Scheduler schedulerFinish = new Scheduler("Dat chizz", true, "09:09");
-        Scheduler schedulerCancel = new Scheduler("Go Yun Pio", false, "00:00");
-        SchedulerSection section = new SchedulerSection();
-        /**
-         * make a section
-         */
-        section.setTitle(title);
-        schedulers.add(schedulerFinish);
-        schedulers.add(schedulerCancel);
-        schedulers.add(schedulerFinish);
-        section.setSections(schedulers);
-        sections.add(section);
-        sections.add(section);
+    public void getSchedulers(String filterChoice, int page, int perpage, String status,
+                              String startDate, String endDate, String monthInput,
+                              String weekInput) {
+        final List<SchedulerSection> sections = new ArrayList<>();
+        final SchedulerSection section = new SchedulerSection();
+        final List<Scheduler> schedulers = new ArrayList<>();
```

Comment:
ERROR: [STYLE][DLS_DEAD_LOCAL_STORE] Dead store to schedulers in com.framgia.fsalon.screen.scheduler.SchedulerPresenter.getSchedulers(String, int, int, String, String, String, String, String)
http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE